app/utils/routing.js

```diff
@@ -65,6 +68,7 @@ export const buildRoute = (pattern, params) => {
         // - '/path(/:param/:another_param)'
         const paramRegex = new RegExp('(/|\\(|\\)|^):' + paramName + '(/|\\)|\\(|$)');
         routePath = routePath.replace(paramRegex, (match, g1, g2) => {
+          // $FlowFixMe
```

Comment:
This line, and the one in tabManager seem to be the only FlowFixMe comments without an explanation.

Reply:
because this one is tricky to get working correctly so I didn't want to spend time on it whereas the other places it's actually Flow that is wrong and so I just `$FlowFixMe` to ignore it.

Reply:
Oh, I meant just adding a comment like "$FlowFixMe : this one is actually tricky because (...)", given that you already know why it's tricky, so the rest of us can have a hint of what's going on. For example, the one about chrome: "$FlowFixMe : flow complains about chrome not being declared".  🙏